node/p2p/blossomsub.go

```diff
@@ -646,21 +644,12 @@ func monitorPeers(ctx context.Context, logger *zap.Logger, h host.Host) {
 		case <-ctx.Done():
 			return
 		case <-time.After(period):
-			// This is once again a snapshot of the peers at the time of the ping. If new peers
+			// This is once again a snapshot of the connected peers at the time of the ping. If new peers
 			// are added between the snapshot and the ping, they will be pinged in the next iteration.
 			peers := h.Network().Peers()
-			connected := make([]peer.ID, 0, len(peers))
```

Comment:
`Peers()` already returns connections which are the connected or limited state, so the filtering was superfluous.